lib/nationbuilder/client.rb

```diff
@@ -44,6 +45,8 @@ def raw_call(path, method, body = {}, args = {})
       }
     }
 
+    request_args[:query][:fire_webhooks] = @fire_webhooks unless @fire_webhooks.nil?
```

Comment:
Let's not handle `fire_webhooks` here. Let's handle it in the `#call` method as a special case if it's passed in as an argument.